src/Tools/EntityGenerator.php

```diff
@@ -1408,21 +1421,26 @@ protected function generateEntityStubMethod(ClassMetadataInfo $metadata, $type,
     }
 
     /**
-     * @param string $name
-     * @param string $methodName
+     * @param string[] $events
      *
      * @return string
      */
-    protected function generateLifecycleCallbackMethod($name, $methodName, ClassMetadataInfo $metadata)
+    protected function generateLifecycleCallbackMethod(array $events, string $methodName, ClassMetadataInfo $metadata)
```

Comment:
This is a breaking change for extending classes. Please revert this part and resort to detecting the type of `$name` to decide what to do inside the method.

Reply:
Will do. Thank for the guidance.